.env.sample

```diff
@@ -38,3 +38,6 @@ LOOKERSDK_CLIENT_SECRET=XXX_REPLACE_WITH_YOUR_CLIENT_SECRET
 LOOKERSDK_VERIFY_SSL=true
 # Timeout in seconds for HTTP requests. Defaults to 2 minutes (120) seconds if not specified.
 LOOKERSDK_TIMEOUT=120
+
+# Threshold for hiding messages with small impression numbers
+IMPRESSIONS_THRESHOLD=1000
```

Comment:
Maybe HIDDEN_MESSAGE_IMPRESSION_THRESHOLD for this and the similar change to variables related to it to make it a bit clearer what this is for when one doesn't have all the context?